src/test/java/jenkins/plugins/http_request/HttpRequestStepTest.java

```diff
@@ -0,0 +1,499 @@
+package jenkins.plugins.http_request;
+
+import hudson.model.Result;
+
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import jenkins.plugins.http_request.auth.BasicDigestAuthentication;
+import jenkins.plugins.http_request.auth.FormAuthentication;
+import jenkins.plugins.http_request.util.NameValuePair;
+import jenkins.plugins.http_request.util.RequestAction;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.http.HttpHost;
+
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+
+import org.junit.Test;
+import static org.junit.Assert.assertTrue;
+
+public class HttpRequestStepTest extends HttpRequestTestBase {
+
+    @Test
+    public void simpleGetTest() throws Exception {
+        // Prepare the server
+        final HttpHost target = start();
+        final String baseURL = "http://localhost:" + target.getPort();
+
+        // Configure the build
+        WorkflowJob proj = j.jenkins.createProject(WorkflowJob.class, "proj");
+        proj.setDefinition(new CpsFlowDefinition(
+            "def response = httpRequest '"+baseURL+"/doGET'\n" +
+            "println('Status: '+response.getStatus())\n" +
+            "println('Response: '+response.getContent())\n",
+            true));
+
+        // Execute the build
+        WorkflowRun run = proj.scheduleBuild2(0).get();
+
+        // Check expectations
+        j.assertBuildStatusSuccess(run);
+        j.assertLogContains("Status: 200",run);
+        j.assertLogContains("Response: "+allIsWellMessage,run);
+    }
+
+    @Test
+    public void canDetectActualContent() throws Exception {
+        // Setup the expected pattern
+        String findMe = allIsWellMessage;
+
+        // Prepare the server
+        final HttpHost target = start();
+        final String baseURL = "http://localhost:" + target.getPort();
+
+        // Prepare HttpRequest
+        HttpRequest httpRequest = new HttpRequest(baseURL+"/doGET");
```

Comment:
I do not understand. Are you talking about the test `canDetectActualContent`, or line number 61?